plugins/vendor/webex/batch/processors/KWebexAPIDropFolderEngine.php

```diff
@@ -347,31 +392,74 @@ protected function createEntryDescriptionFromRecording($dropFolderFile)
 	}
 	
 	/**
-	 * @param KalturaMediaEntry $entry
-	 * @param array $validatedUsers
-	 * @throws kCoreException
+	 * @return array|null
 	 */
-	protected function handleParticipants($entry)
+	protected function getAdditionalUsers($ownerId)
 	{
-		//$validatedUsers = $this->getAdditionalUsers($this->dropFolderFile->meetingId);
-		$validatedUsers = array();
+		$meetingId = $this->dropFolderFile->meetingId;
+		$hostEmail = $this->dropFolderFile->hostEmail;
+		$participantsList = $this->webexClient->getMeetingParticipants($meetingId, $hostEmail);
+		if (!isset($participantsList['items']))
+		{
+			KalturaLog::warning("Error getting meeting participants from Webex for meeting id [$meetingId], response: " . print_r($participantsList, true));
+			throw new kApplicativeException(KalturaDropFolderErrorCode::DROP_FOLDER_APP_ERROR, DropFolderPlugin::MISSING_MEETING_PARTICIPANTS_INFO);
+		}
 		
-		$handleParticipantMode = $this->dropFolder->webexAPIVendorIntegration->handleParticipantsMode;
-		if ($validatedUsers && $handleParticipantMode != kHandleParticipantsMode::IGNORE)
+		$userToExclude = strtolower($ownerId);
+		$additionalWebexUsers = $this->parseAdditionalUsers($participantsList['items']);
+		return $this->getValidatedUsers($additionalWebexUsers, $this->dropFolder->partnerId, $this->dropFolder->webexAPIVendorIntegration->createUserIfNotExist, $userToExclude);
+	}
+	
+	protected function parseAdditionalUsers($additionalUsersWebexResponse)
+	{
+		$usersList = array();
+		
+		foreach ($additionalUsersWebexResponse as $user)
 		{
-			switch ($handleParticipantMode)
+			if (!isset($user['email']) || !isset($user['host']))
 			{
-				case kHandleParticipantsMode::ADD_AS_CO_PUBLISHERS:
-					$entry->entitledUsersPublish = implode(',', array_unique($validatedUsers));
-					break;
-				case kHandleParticipantsMode::ADD_AS_CO_VIEWERS:
-					$entry->entitledUsersView = implode(',', array_unique($validatedUsers));
-					break;
-				case kHandleParticipantsMode::IGNORE:
-				default:
-					break;
+				KalturaLog::warning('Error getting information for participant, participant details: ' . print_r($user, true));

```

Comment:
if we throw it here we will crash the entire upload process for the partner, if its just participants and not the owner i think we should just print the error